src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -2840,7 +2840,8 @@ private void ReinferMethodAndVisitArguments(BoundCall node, TypeWithState receiv
             }
 
             ImmutableArray<VisitArgumentResult> results;
-            (method, results) = VisitArguments(node, node.Arguments, refKindsOpt, method.Parameters, node.ArgsToParamsOpt,
+            ImmutableArray<ParameterSymbol> notNullParameters;
+            (method, results, notNullParameters) = VisitArguments(node, node.Arguments, refKindsOpt, method.Parameters, node.ArgsToParamsOpt,
```

Comment:
>notNullParameters [](start = 30, length = 17)

Consider returning a single `bool` indicating whether the return value is not null.

Reply:
I don't think we can do that. We need to know which parameters are not null so that we can apply `NotNullIfNotNull` on parameters as post-conditions (ie. after visiting the arguments).